### PR TITLE
{bio}[foss/2021a] AUGUSTUS v3.4.0

### PR DESCRIPTION
**Pull request by SIB-software**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.4.0-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Updated by: Sebastien Moretti (SIB)
+
+easyblock = 'ConfigureMake'
+
+name = 'AUGUSTUS'
+version = '3.4.0'
+
+homepage = 'https://bioinf.uni-greifswald.de/augustus/'
+description = "AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences"
+# software_license = Artistic
```
Comment (jfgrimm) [inline easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.4.0-foss-2021a.eb:10]: consider adding a corresponding license entry to: https://github.com/easybuilders/easybuild-framework/blob/develop/easybuild/framework/easyconfig/licenses.py

```suggestion
# software_license = 'LicenseArtisticv1'
```

File: easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.4.0-foss-2021a.eb [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Updated by: Sebastien Moretti (SIB)
```
Comment (jfgrimm) [inline easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.4.0-foss-2021a.eb:1]: please keep the previous attribution
```suggestion
# Updated by: Pavel Grochal (INUITS)
# Updated by: Sebastien Moretti (SIB)
# License: GPLv2
```

File: easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.4.0-foss-2021a.eb (hunk 1 above)
Comment (smoretti) [reply easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.4.0-foss-2021a.eb]: done

File: easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.4.0-foss-2021a.eb (hunk 2 above)
Comment (smoretti) [reply easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.4.0-foss-2021a.eb:2]: Done
I did not keep `License: GPLv2` because this is not the Augustus license

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node125.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/f85ff88c041ce01904d5b7ff42bc85c0 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in total)
centos7.lan - Linux CentOS Linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/3463c9f3392c6ea2228cda5d7c7465ef for a full test report.

**Comment by jfgrimm** [comment]
@boegelbot please test @ jsc-zen2

**Review by verdurin — APPROVED** [review]
Looks fine.

**Pull request merged**